client/common/js/collapsibleCard.js

```diff
@@ -0,0 +1,30 @@
+function collapsibleCard(card) {
+	function toggleCard() {
+		if (card.classList.contains('closed')) {
+			card.classList.remove('closed')
+			card.classList.add('open')
+			card.querySelectorAll('.incident-card-collapsible__toggle-button-text')
+				// eslint-disable-next-line no-param-reassign
+				.forEach((buttonText) => { buttonText.innerText = 'Show Less' })
```

Comment:
I think we also need to update the toggle  icon from a plus to a cross and vice-versa in the other scenario. Also, do we really need to `querySelectorAll`? I don't think there will ever be a case where there will be more than one "Show more" button, and that way we can avoid the `forEach`?